src/helpers/assetHelper.ts

```diff
@@ -0,0 +1,19 @@
+import { Asset } from '@thorchain/asgardex-util'
+import { sequenceS } from 'fp-ts/lib/Apply'
+import * as FP from 'fp-ts/lib/function'
+import * as O from 'fp-ts/lib/Option'
+
+export const unsafeAssetToString = (asset: Asset): string => {
+  const { chain, symbol } = asset
+  return `${chain}.${symbol}`
+}
+
+const sequenseSO = sequenceS(O.option)
```

Comment:
maybe its better to move such things to some 'fp-helpers' ?
i suppose sequenseSO or similar may be needed across the project

Reply:
Good suggestion!

Reply:
We will keep that in mind. For now we don't need to extract `sequenseSO` into a `fp-helpers`, since `assetToString` [will be moved into `@thorchain/asgardex-util` package](https://gitlab.com/thorchain/asgardex-common/asgardex-util/-/issues/9).